Source/Game/Discord/DiscordMessageQueue.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Game.Network.Packets;
+
+namespace Game.Discord
+{
+    public static class DiscordMessageQueue
+    {
+        public static readonly List<DiscordMessage> DiscordMessages = new List<DiscordMessage>();
```

Comment:
You're probably better off using something like a `BlockingCollection<T>` here, as it will provide clearer semantics when multiple threads use it, as is likely to happen with a singleton like this.

Reply:
I have to study BlockingCollection xD

Reply:
https://docs.microsoft.com/en-us/dotnet/standard/collections/thread-safe/blockingcollection-overview